packages/app/src/cli/services/init/init.ts

```diff
@@ -197,8 +198,10 @@ async function init(options: InitOptions) {
 
   let app: OrganizationApp
   if (options.selectedAppOrNameResult.result === 'new') {
-    // Load the local app to get the creation options. No need for specs since we only care about Creation Options.
-    const localApp = await loadApp({specifications: [], directory: outputDirectory, userProvidedConfigName: undefined})
+    // Load the local app to get the creation options.
+    // NOTE: need the specs to load the scopes, if we ever remove the local specs, we need to find a way to do this without them.
+    const specifications = await loadLocalExtensionsSpecifications()
```

Comment:
does this load the entire existing TOML of the app?  Is there any chance this could break things when we use `app init` just by itself?